### ISSUE TITLE
PyYAML 5.4 fails to build with cython error on Raspbian

### ISSUE THREAD

**Issue by serversonic** [PowerDNS-Admin/PowerDNS-Admin#1665]
### PDA version

0.4.1

### Python version

3.9

### Steps to Reproduce

Run `pip install -r requirements.txt`
PyYAML fails with 

```
        raise AttributeError(attr)
      AttributeError: cython_sources
```

### Expected Behavior

PyYAML builds correctly and the pip install continues

### Observed Behavior

```
        raise AttributeError(attr)
      AttributeError: cython_sources
```

**Comment by serversonic** [comment]
Adding this as a note with a solution.

Replacing `PyYAML=5.4` with `PyYAML=5.3.1` works, as does `PyYAML=6.0.1` (current version).

**Comment by vdchuyen** [comment]
yeah, same on mac brew: https://github.com/yaml/pyyaml/issues/724 due to Cython 3.0

**Comment by AzorianMatt** [comment]
This has likely been resolved by my update of the project requirements to use PyYAML 6.0.1. The change is now present in the `dev` branch of the project.
